src/labeler.ts

```diff
@@ -122,10 +127,10 @@ async function fetchContent(
   return Buffer.from(response.data.content, response.data.encoding).toString();
 }
 
-function getLabelGlobMapFromObject(
+function getLabelConfigMapFromObject(
   configObject: any
-): Map<string, StringOrMatchConfig[]> {
-  const labelGlobs: Map<string, StringOrMatchConfig[]> = new Map();
+): Map<string, MatchConfig[]> {
+  const labelGlobs: Map<string, MatchConfig[]> = new Map();
   for (const label in configObject) {
     if (typeof configObject[label] === 'string') {
       labelGlobs.set(label, [configObject[label]]);
```

Comment:
I think we can get rid of this condition since a simple string (`label: 'pattern'`) is no longer a valid glob. We should throw the error in this case.